react/features/base/media/components/native/VideoTransform.js

```diff
@@ -0,0 +1,732 @@
+// @flow
+
+import React, { Component } from 'react';
+import { PanResponder, View } from 'react-native';
+import { connect } from 'react-redux';
+
+import { storeVideoTransform } from '../../actions';
+
+import styles from './styles';
+
+/**
+ * The default/initial transform (= no transform).
+ */
+const defaultTransform = {
```

Comment:
How is `defaultTransform` different from `MIN_SCALE`, etc. which you converted from `minScale`? Is it not being used as a literal?

Reply:
The MIN_SCALE just happens to be equal to defaultTransform.scale for now, but they are not related. We can reduce MIN_SCALE to below 1 (if we want to allow the user to be able to scale under 1x magnification) but the defaultTransform.scale will still remain 1, as that means no-op.

Reply:
I'm saying that 'defaultTransform' appears to be a "literal" just line "MIN_SCALE" so I don't understand why you applied the appropriate naming to "MIN_SCALE" but didn't change `defaultTransform` to `DEFAULT_TRANSFORM`. Am I missing something?

Reply:
Ah, sorry, I misunderstood then. Yeah, I fix that, thanks!